cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,21 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
-
-  });
+  beforeEach(() => {
+ cy.visit('https://react-redux.realworld.io')
+});
+it('should login user', () => {
+ const randomNumber = Math.random().toString().slice(2);
```

Comment:
The syntax for template literals is incorrect. Use backticks instead of single quotes for defining 'userName'.

Reply:
Not fixed

Reply:
Not fixed